web/src/peer_data.ts

```diff
@@ -45,7 +64,7 @@ export async function maybe_fetch_stream_subscribers(stream_id: number): Promise
             const xhr = await channel.get({
                 url: `/json/streams/${stream_id}/members`,
             });
-            subscribers = fetch_stream_subscribers_response_schema.parse(xhr).subscribers;
+            subscribers = fetch_stream_subscribers_response_schema.parse(await xhr).subscribers;
```

Comment:
This does nothing: `xhr` is already the result of `await`ing a promise above (and thus is misnamed: `channel.get(…)` is a `jqXHR` object, but `await channel.get(…)` is just the response data).

Reply:
Oh shoot thanks for catching that!

#34562